src/NumericalAlgorithms/Spectral/Spectral.cpp

```diff
@@ -246,12 +246,28 @@ const DataVector& quadrature_weights(const size_t num_points) noexcept {
       .second;
 }
 
-template <Basis BasisType, Quadrature QuadratureType>
-const Matrix& differentiation_matrix(const size_t num_points) noexcept {
-  return precomputed_spectral_quantity<
-      BasisType, QuadratureType,
-      DifferentiationMatrixGenerator<BasisType, QuadratureType>>(num_points);
-}
+/// \cond
+// clang-tidy: Macro arguments should be in parentheses, but we want to append
+// template parameters here. Neither NOLINT nor NOLINTNEXTLINE can suppress this
+// warning.
```

Comment:
This clang-tidy warning arises since `generator_name` below is not in parentheses, and as described here I can't silence it. Any suggestions?

Reply:
Does something like [this](https://github.com/sxs-collaboration/spectre/blob/develop/src/Utilities/FakeVirtual.hpp#L73) work?

Reply:
That works, thanks for the pointer! 👍